crates/l2/contracts/src/l1/interfaces/IOnChainProposer.sol

```diff
@@ -23,6 +23,10 @@ interface IOnChainProposer {
     /// @dev Event emitted when a batch is verified.
     event BatchVerified(uint256 indexed lastVerifiedBatch);
 
+    /// @notice Set the bridge address for the first time.
+    /// @param bridge the address of the bridge contract.
```

Comment:
Explain why we need this separate for `@dev`.

Reply:
Done [d65e021](https://github.com/lambdaclass/ethrex/pull/2809/commits/d65e021f4e8a6b007c066264c24066f260844371)